neuralcoref/train/training.md

```diff
@@ -43,9 +43,9 @@ or by following these steps:
    * `cat conll-2012/v4/data/train/data/my_lang/annotations/*/*/*/*.v4_gold_conll >> train.my_lang.v4_gold_conll`
    * `cat conll-2012/v4/data/development/data/my_lang/annotations/*/*/*/*.v4_gold_conll >> dev.my_lang.v4_gold_conll`
    * `cat conll-2012/v4/data/test/data/my_lang/annotations/*/*/*/*.v4_gold_conll >> test.my_lang.v4_gold_conll`
-   
+
 ## Prepare the data
-Once you have the set of `*.v4_gold_conll` files, you can prepare the training data by running 
+Once you have the set of `*.v4_gold_conll` files, move these files into seperate (`train`, `test`, `dev`) subdirectories inside `data` directory. Now, you can prepare the training data by running 
 [conllparser.py](/neuralcoref/train/conllparser.py) on each split of the data set (`train`, `test`, `dev`) as
```

Comment:
this doesn't necessarily need to be the same `data` directory than the one(s) in your `conll-2012` folder, so maybe we should make that explicit and call it `my_data` orso instead: 
> ... inside a new `my_data` directory

(also adjust in the bash script lines)

Reply:
does this work?